modules/sum-transformers/additional/models/models.go

```diff
@@ -0,0 +1,19 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package models
+
+// Summary used in sum module to represent
+// the summary to a given content
+type Summary struct {
```

Comment:
I could not found where this struct is used.
Anyway if it is the model It make sense to me to move it to the upper level package.

Reply:
I think that this struct can be actually removed, it's not used anywhere

Reply:
Done